src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java

```diff
@@ -11,10 +11,10 @@
 
     @Schema(description = "The key of the field that the validation error occurred", example = "userName")
     private String field;
-    @Schema(description = "Description of the error that occurred", example = "must not be empty")
+    @Schema(description = "Description of the error that occurred", example = "must have at least 3 characters")
     private String message;
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @Schema(description = "The value that was passed to this field", example = "invalidValue")
+    @Schema(description = "The value that was passed to this field", example = "dé")
```

Comment:
dé? 🧐

Reply:
Nickname of a person, example just a example that the field must at least contains 3 chars

Reply:
dé? 🧑‍🚀